conf/psc/openupgrade.yml

```diff
@@ -13,5 +13,6 @@ openupgrade-maintainers:
 openupgrade-maintainers-psc-representative:
   members:
     - StefanRijnhart
+    - legalsylvain
```

Comment:
Please include me to be able also to handle it, or I think I will lose it the current right when synced

Reply:
I'm not sure how this repo works, but I _think_ that as you are member of the core team (https://github.com/OCA/repo-maintainer-conf/blob/master/conf/psc/core.yml) you have very broad rights granted by this "global rule (https://github.com/OCA/repo-maintainer-conf/blob/master/conf/global.yml)

Anyway updated.

Reply:
add me too :)

Reply:
done.